guardian/prosemirror-typerighter
# Update build script to include Typescript definitions

## What does this change?
This PR updates the build script to include Typescript definitions.

## How to test

Run build locally and link in third-party project. Plugin should work as expected.

## How can we measure success?

Typerighter plugin works when linked via `npm link`.
